### PR TITLE
initial README

### LINKED ISSUE #19: Initial README
/kind documentation

**Describe the solution you'd like**
Add something to the README. Keep it short and informative. Do not use many words. Consider commenting on this issue about ideas for future documentation that we might keep in a docs directory.

/cc @amy

### PR DESCRIPTION
**Pull request by fabriziopandini**
Something easy to get me started on this 😉 
Fixes #19

@chuckha I made some assumptions for Versioning, Maintenance, and Compatibility; if it is too early for making decisions about this, I can remove this paragraph

/assign @chuckha 
/cc @amy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,56 @@
-# Kubernetes Template Project
+# Cluster API bootstrap provider kubeadm
+## What is the Cluster API bootstrap provider kubeadm?
 
-The Kubernetes Template Project is a template for starting new projects in the GitHub organizations owned by Kubernetes. All Kubernetes projects, at minimum, must have the following files:
+Cluster API bootstrap provider is a component of
+[Cluster API](https://github.com/kubernetes-sigs/cluster-api/blob/master/README.md) 
+that is responsible of generating a cloud-init script to
+turn a Machine into a Kubernetes node; this implementation uses [kubeadm](https://github.com/kubernetes/kubeadm)
```
Comment (detiber) [inline README.md]:
```suggestion
turn a Machine into a Kubernetes Node; this implementation uses [kubeadm](https://github.com/kubernetes/kubeadm) 
```

**Comment by fabriziopandini** [comment]
@chuckha comments addressed

File: README.md [hunk 2]
```diff
@@ -1,29 +1,56 @@
...
 
 ### Code of conduct
 
 Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
 
 [owners]: https://git.k8s.io/community/contributors/guide/owners.md
 [Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
+
+### Useful links
+
```
Comment (chuckha) [inline README.md]: The only problem is now this sections is showing up as empty. you can reuse the footnote links here!

```
* [Meeting notes][notes]
* [Meeting recordings][recordings]
```
and so on

Comment (chuckha) [inline README.md]: The only problem is now this sections is showing up as empty. you can reuse the footnote links here!

* [Meeting notes][notes]
* [Meeting recordings][recordings]
and so on

**Comment by fabriziopandini** [comment]
I remove Useful Link section and also code of conducts (I noticed there is a separated file for this)

**Comment by chuckha** [comment]
thank you @fabriziopandini !! Welcome aboard :)

**Pull request merged**